.travis.yml

```diff
@@ -6,7 +6,9 @@ php:
   - 5.6
   - 5.5
   - 5.4
-  - hhvm
+  - hhvm-3.30
+  - hhvm-3.27
+  - hhvm-3.24
```

Comment:
3.24 ended support on 2018-12-17, but I figure we can go ahead and keep running the tests against it as long as it still works. See https://docs.hhvm.com/hhvm/installation/release-schedule

Reply:
Please add it to `allow_failures` list